app/controllers/detail-viewers.ts

```diff
@@ -171,12 +201,52 @@ function getDocumentViewer(documentManagementService: DocumentManagementService)
   };
 }
 
-function setupDetailViewersController(middleware: Middleware[], documentManagementService: DocumentManagementService): Router {
+function getTimeExtensionViewer(req: Request, res: Response, next: NextFunction) {
+  try {
+    const timeExtensionId = req.params.id;
+    const timeExtensionData: TimeExtensionCollection = timeExtensionIdToTimeExtensionData(timeExtensionId, req.session.appeal.timeExtensionEventsMap);
+    if (timeExtensionData) {
+      let previousPage: string = paths.common.overview;
+      const data = setupTimeExtension(req, timeExtensionData);
+      return res.render('detail-viewers/time-extension-details-viewer.njk', {
+        previousPage: previousPage,
+        data: data
+      });
+    }
+    // SHOULD THROW NOT FOUND
```

Comment:
Is this a todo?

Reply:
Yup waiting on RIA-2813: File not found page